packages/cli/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@bull-monitor/cli",
+  "version": "2.12.0",
+  "description": "cli for bull-monitor",
+  "keywords": [
+    "bull"
+  ],
+  "author": "Ilya Strus <strax1990@gmail.com>",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/s-r-x/bull-monitor.git",
+    "directory": "packages/cli"
+  },
+  "license": "MIT",
+  "types": "dist/index.d.ts",
+  "bin": {
+    "semver": "dist/index.js"
```

Comment:
Is there any reason why the command is called semver?

Reply:
@s-r-x it was typo. corrected it.